app/Providers/AppServiceProvider.php

```
@@ -21,7 +22,7 @@ public function register()
      */
     public function boot()
     {
-        if(config('app.env') != "local"){
+        if (config('app.env') != "local") {
             URL::forceScheme('https');
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\URL' in method 'boot'. (undefined)

(StaticAccess)